docs/index.md

```diff
@@ -104,7 +104,7 @@ To allow Firebase on iOS to use the credentials, the Firebase iOS SDK must be co
 
 To do this, open your `/ios/{projectName}/AppDelegate.mm` file (or `AppDelegate.m` if on older react-native), and add the following:
 
-At the top of the file, import the Firebase SDK:
+At the top of the file, import the Firebase SDK right after `'#import "AppDelegate.h"'`:
```

Comment:
This will likely help people who mistakenly put it inside a compiler IFDEF conditional, that happens frequently. Good suggestion